draft-yoshino-wish.txt

```diff
@@ -252,11 +252,12 @@ Internet-DraWiSH: A General Purpose Message Framing over Byt  March 2017
    underlying protocols.
 
    The permessage-deflate extension [RFC7692] is defined for the
-   WebSocket protocol, to add a compression mechanism to it.  No
-   extension mechanism is defined for WiSH.  Compression can be
-   implemented by underlying protocols or in the application layer if
-   needed.  What contents are exchanged and in what encoding they are
-   exchanged over WiSH are to be defined by the application layer.
+   WebSocket protocol, to add a compression mechanism to it.  The
+   permessage-deflate extension can be applied to WiSH.  The details are
+   to be specified [BidiwebCompression].
```

Comment:
I don't think we should link to any github issues from I-D. 

"The permessage-deflat extension defined for WS can be applied to the WiSH framing. Full body compression may also be enabled, which is transparent to the WiSH framing. "

p.s. when "WiSH" is mentioned in the I-D, we should qualify it with specific meaning, e.g. WiSH framing etc What do you think?